Sming/Libraries/ModbusMaster/samples/generic/include/application.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
That header file is not needed. Remove it and also remove its reference in application.cpp.

Reply:
Well I placed the callback functions above init() in application.cpp to sucessfully compile it without the header file.